packages/calcite-components/src/components/flow-item/flow-item.scss

```diff
@@ -3,26 +3,64 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-flow-item-footer-padding: Specifies the padding of the component's footer.
- * @prop --calcite-flow-item-header-border-block-end: Specifies the component header's block end border.
+ * @prop --calcite-flow-item-background-color: Specifies the background color of the component.
+ * @prop --calcite-flow-item-border-color: Specifies the border color of the component.
+ * @prop --calcite-flow-item-description-text-color: Specifies the text color of the component's description.
+ * @prop --calcite-flow-item-fab-z-index: Specifies the component fab's z-index.
+ * @prop --calcite-flow-item-footer-background-color: Specifies the background color of the component's footer.
+ * @prop --calcite-flow-item-footer-padding: [Deprecated] Use `--calcite-flow-item-footer-space` instead. Specifies the padding of the component's footer.
+ * @prop --calcite-flow-item-footer-space: Specifies the spacing of the component's footer.
+ * @prop --calcite-flow-item-header-background-color: Specifies the component header's background color.
+ * @prop --calcite-flow-item-header-border-block-end: [Deprecated] No longer necessary. Specifies the component header's block end border.
+ * @prop --calcite-flow-item-heading-text-color: Specifies the component's heading text color.
+ * @prop --calcite-flow-item-header-z-index: Specifies the component header's z-index.
  */
 
 :host {
-  @extend %component-host;
-  @apply relative flex w-full flex-auto;
+  @include base-host();
+  --calcite-flow-item-border-color: var(--calcite-color-border-3);
+  --calcite-flow-item-description-text-color: var(--calcite-color-text-2);
+  --calcite-flow-item-fab-z-index: var(--calcite-z-index-sticky);
+  --calcite-flow-item-footer-background-color: var(--calcite-color-foreground-1);
+  --calcite-flow-item-footer-space: var(--calcite-flow-item-footer-padding, var(--calcite-spacing-xxs));
+  --calcite-flow-item-heading-text-color: var(--calcite-color-text-2);
+  --calcite-flow-item-header-background-color: var(--calcite-color-foreground-1);
+  --calcite-flow-item-header-z-index: var(--calcite-z-index-header);
+  --calcite-flow-item-header-border-width: var(--calcite-border-width-sm);
+  --calcite-flow-item-header-border-block-end: var(--calcite-border-width-sm) solid
+    var(--calcite-flow-item-border-color);
```

Comment:
Should we even set these defaults or just let them be handled at the internal `calcite-panel` level?

Reply:
I think setting defaults here is good for the time being given the current construction of the component.